x/ratelimit/keeper/rate_limit_test.go

```diff
@@ -288,11 +292,14 @@ func (s *KeeperTestSuite) processCheckRateLimitAndUpdateFlowTestCase(tc checkRat
 			continue
 		}
 
-		if action.addToWhitelist != "" {
-			s.App.RatelimitKeeper.AddAddressToWhitelist(s.Ctx, action.addToWhitelist)
+		if action.addToWhitelist {
```

Comment:
Do we check somewhere that if A-> B is whitelisted, then A-> C and C-> B both get rate limited?

Reply:
it's covered in `TestAddressWhitelist`!